.github/workflows/scripts/push-scratch-images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+IMAGETAG="$1"
+if [ -z "$IMAGETAG" ]; then
+    echo "IMAGETAG not provided!" 1>&2
+    echo "Usage: push-images.sh IMAGETAG" 1>&2
+    exit 1
+fi
+
+# Extracting org name rather than hardcoding allows this
+# action to be portable across forks
+ORGNAME=`echo $GITHUB_REPOSITORY | tr '/' "\n" | head -1 | tr -d "\n"`
+
+echo "Pushing images tagged as $IMAGETAG..."
+
+for img in spire-server spire-agent; do
```

Comment:
Notably missing here is `k8s-workload-registrar` and `oidc-discovery-provider`.

The GHCR repo is namespaced under the org rather than the repo, unfortunately. This means we need to be extra careful when selecting image names ... they should clearly reflect their purpose and make sense under the larger SPIFFE scope. In my opinion, these two names do not meet that requirement.

For the `k8s-workload-registrar` ... we will soon have new spire-controller code being published as an image on GHCR, so I kind of feel like we should just roll with that. It is already namespaced appropriately.

For the `oidc-discovery-provider`.. it relies on SPIRE APIs. I'm not sure what's best ... maybe `spire-oidc-provider`? Or `spire-oidc-discovery-provider`?

For now I've left these two out.

Reply:
spire-oidc-provider seems ok?